chain/index/ddls_test.go

```diff
@@ -0,0 +1,708 @@
+package index
+
+import (
+	"database/sql"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestHasRevertedEventsInTipsetStmt(t *testing.T) {
+	s, err := NewSqliteIndexer(":memory:", nil, 0, false, 0)
+	require.NoError(t, err)
+
+	// running on empty DB should return false
+	verifyHasRevertedEventsInTipsetStmt(t, s, []byte("test_tipset_key"), false)
```

Comment:
lots of use of these byte strings, how about you extract them to local consts in this file

Reply:
Done.